examples/generic_connector_configs/README.md

```diff
@@ -150,6 +151,49 @@ The configuration file for the GitHub API can be found at [github_secretless.yml
 
 ___
 
+### Facebook API
+This example focuses on [Instagram](https://developers.facebook.com/docs/instagram), but can be
+ used to interact with [Facebook's many API's](https://developers.facebook.com/docs/instagram).
```

Comment:
This is the same link as the one above. Can we be more explicit about what APIs from Facebook this supports?

Reply:
Attached the wrong link, good catch!